native/cocos/bindings/jswrapper/v8/Object.h

```diff
@@ -51,6 +51,35 @@ namespace se {
 class Class;
 class ScriptEngine;
 
+class MapOperation final {
+public:
+    MapOperation(Object *obj);
+
+    void clear();
+    bool remove(const ccstd::string &key);
+    bool get(const ccstd::string &key, Value *outValue);
+    bool set(const ccstd::string &key, const Value &value);
+    uint32_t getSize() const;
+    ccstd::vector<std::pair<std::string, Value>> getAll() const;
```

Comment:
Is there any reason to return a vector of pair instead of a map?

Reply:
I think it could be a `std::unordered_map`. 
The reason why I uses vector is that V8 API has a `AsArray()` method to get all keys and values :
```c++
class V8_EXPORT Map : public Object {
 public:
  size_t Size() const;
  void Clear();
  V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                              Local<Value> key);
  V8_WARN_UNUSED_RESULT MaybeLocal<Map> Set(Local<Context> context,
                                            Local<Value> key,
                                            Local<Value> value);
  V8_WARN_UNUSED_RESULT Maybe<bool> Has(Local<Context> context,
                                        Local<Value> key);
  V8_WARN_UNUSED_RESULT Maybe<bool> Delete(Local<Context> context,
                                           Local<Value> key);

  /**
   * Returns an array of length Size() * 2, where index N is the Nth key and
   * index N + 1 is the Nth value.
   */
  Local<Array> AsArray() const;

......

 private:
  Map();
  static void CheckCast(Value* obj);
};
```

Reply:
I think it could not be std::map or std::unordered_map since there is not a compare method for `se::Value`.
If we return std::map, it will generate a compile error.
```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.0.sdk/usr/include/c++/v1/__functional/operations.h:487:21: error: invalid operands to binary expression ('const se::Value' and 'const se::Value')
        {return __x < __y;}
```

Compiler doesn't know how to compare two `se::Value`s.